migrations/0000_createDbs.sql

```diff
@@ -29,7 +29,7 @@ CREATE TABLE IF NOT EXISTS note_to_publication_relationship (
 	publication_id bigint references publication(id) NOT NULL
 );
 
-CREATE TABLE IF NOT EXISTS note_to_type_relationship (
+CREATE TABLE IF NOT EXISTS note_to_category_relationship (
 	note_id bigint PRIMARY KEY references note(id),
-	type note_type NOT NULL
+	type category NOT NULL
```

Comment:
the column name shouldn't be `type`. It feels like it should be `category`, but that is clearly problematic. What about renaming the type to `category_type` or `note_category`?